com.tobagin.whakarere.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.tobagin.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+#rename-desktop-file: whakarere.desktop
+#rename-icon: whakarere.png
+
+finish-args:
+  - --share=ipc     # D-Bus
+  - --share=network  # If your script needs network
+  - --socket=x11    # If your script has a GUI
+  - --socket=wayland # If your script has a GUI
+  - --socket=pulseaudio # If your script needs sound
+  - --socket=session-bus # If your script needs D-Bus
```

Comment:
this is not allowed. You need to determine what dbus name you need.

Reply:
I have remove session-bus entirely, I was going to use for something  but at this stage it will remain without it.